playground/infrastructure/test_helper.py

```diff
@@ -13,12 +13,163 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from api.v1.api_pb2 import SDK_UNSPECIFIED, STATUS_UNSPECIFIED
-from helper import find_examples, Example
+import mock
 
+from unittest.mock import mock_open
+from api.v1.api_pb2 import SDK_UNSPECIFIED, STATUS_UNSPECIFIED, SDK_JAVA
+from helper import find_examples, Example, _get_example, _match_pattern, _get_name, _get_sdk, get_tag, _validate
 
-def test_find_examples():
-    result = find_examples("")
 
-    assert len(result) == 1
-    assert result[0] == Example("", "", SDK_UNSPECIFIED, "", "", "", STATUS_UNSPECIFIED)
+@mock.patch('helper._get_example')
+@mock.patch('helper._match_pattern')
+@mock.patch('helper.os.path.join')
+@mock.patch('helper.os.walk')
+def test_find_examples(mock_os_walk, mock_os_path_join, mock_match_pattern, mock_get_example):
+    example = Example("file", "pipeline_id", SDK_UNSPECIFIED, "root/file.extension", "code", "output",
+                      STATUS_UNSPECIFIED)
+    mock_os_walk.return_value = [("/root", (), ("file.extension",))]
+    mock_os_path_join.return_value = "/root/file.extension"
```

Comment:
Seems like there is no need to mock `os.path.join`

Reply:
Changed.